awscli/argprocess.py

```diff
@@ -245,13 +246,41 @@ def _is_complex_shape(model):
     return True
 
 
+def _uses_old_list_case(service_name, operation_name, argument_name):
+    """
+    Determines whether a given operation for a service needs to use the
+    deprecated shorthand parsing case for lists of structures that only have
+    a single member.
+    """
+    if service_name == 'elb':
+        service_name = 'elasticloadbalancing'
```

Comment:
A little curious. What event (I am assuming that is using endpointPrefix as service name) is this if statement needed for?